WordPress/src/main/res/layout/reader_no_site_to_reblog.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <org.wordpress.android.ui.ActionableEmptyView
```

Comment:
It was not shared before so you could not know but I'm reporting since I noticed while testing this PR: I think we should manage the image of this `ActionableEmptyView` to hide it in landscape so to avoid the action button to not be visible. See the notifications case below:

| ![Screenshot_1586276298](https://user-images.githubusercontent.com/47797566/78699136-b2cf1500-7903-11ea-8957-a5b8a2587585.png) | ![Screenshot_1586276303](https://user-images.githubusercontent.com/47797566/78699148-b82c5f80-7903-11ea-8f05-702d7a08390a.png) |
|---|---|

Reply:
I tried a more generic approach on this since it seems to be an issue in other screens too.
Fix: cab7a8d4c0ca69c4d3ebe9f06406a28ee7dc9f59
I tested that this can also work in cases like the NotificationsListFragmentPage. It fails though in other pages (e.g. MySitesFragment) probably due to the way configuration change is handled. Wdyt?
I can revert this and fix it in NoSiteToReblogActivity only.

Reply:
I agree with your solution here 👍! In the My Site case currently we are conditioning based on the screen height being >= 600 that makes sense to me. I think we need some more digging here to generalize the cases but we can eventually improve the logic by which we hide the image in a dedicated task! For now I'm accepting the modification since it can be generalized and currently has an effect only in the screen we are covering in this PR 👍.